lib/diaspora/mentionable.rb

```diff
@@ -1,23 +1,20 @@
 
 module Diaspora::Mentionable
 
-  # regex for finding mention markup in plain text
-  # ex.
+  # regex for finding mention markup in plain text:
+  #   "message @{user@pod.net} text"
+  # it can also contain a name:
```

Comment:
Nit: `it can also contain a name, which gets used as the link text`, just to be sure.